blank-canvas-blocks/sass/blocks/_heading.scss

```diff
@@ -1,5 +1,5 @@
 h1, h2, h3, h4, h5, h6 {
 	clear: both;
-	font-family: var(--wp--custom--heading--typography--font-family);
+	font-family: var(--wp--custom--fonts--primary);
```

Comment:
I believe this shouldn't change from --heading--typography--font-family so that this can be configured via theme.json specifically.  

If we want to use custom fonts 'primary', 'secondary' or 'pickle' perhaps that should stay in theme.json rather than being consumed directly by any .scss blocks.  Setting the value of --heading--typography--font-family to "var(--wp--custom--fonts-primary)" seems fine though.

Reply:
This change was just for testing purposes